internal/util/indexcgowrapper/index.go

```diff
@@ -339,3 +326,38 @@ func (index *CgoIndex) CleanLocalData() error {
 	status := C.CleanLocalData(index.indexPtr)
```

Comment:
1. seems cleanLocal Data is not actually used in index building path.
2. what happened if clean failed? the garbage data is leaved there, would that affect next index build or load?
3.

Reply:
1. fixed
2. Even the garbage data is leaved there, don't  affect next index build or load. The local index and raw data files will be removed when rebuild or load
 https://github.com/milvus-io/milvus/blob/fce815e4145b279eda6fdf1aa3367153cf0a9e6a/internal/core/src/index/VectorDiskIndex.cpp#L34-L48